app/controllers/expenses_controller.rb

```diff
@@ -4,9 +4,10 @@ def new
 
   def create
     @expense = Expense.new(expense_params)
```

Comment:
You could probably merge these 2 lines into one. Instead of 
```ruby
@expense = Expense.new(expense_params)
@expense.user = current_user
```

You could try 

```ruby
@expense = Expense.new(expense_params.merge(user: current_user))
```

This should add the user field to the expense hash before it creates the new object